includes/Integrations/Events_Calendar/Compatibility/TEC_V6_Compatibility.php

```
@@ -0,0 +1,773 @@
+
+            if ( ! $event_id ) {
+                return new \WP_Error( 'tec_api_exception', 'Failed to create event using ORM API' );
+            }
+
+            $this->logger->log( 'TEC v6: Event created successfully using ORM API with ID ' . $event_id );
+            return true;
+
+        } catch ( \Exception $e ) {
+            return new \WP_Error( 'tec_api_exception', $e->getMessage() );
+        }
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'save_event_using_tec_api'. (Unused Code Rules)

(UnusedPrivateMethod)